libraries/AP_HAL_ChibiOS/hwdef/scripts/STM32H743xx.py

```diff
@@ -20,10 +20,11 @@
     # flags of 4 means memory can be used for SDMMC DMA
     'RAM_MAP' : [
         (0x20000000, 128, 2), # DTCM, tightly coupled, no DMA, fast
-        (0x30000000, 288, 0), # SRAM1, SRAM2, SRAM3
-        (0x38000000,  64, 1), # SRAM4. This supports both DMA and BDMA ops
+        (0x30000000, 272, 0), # SRAM1, SRAM2
```

Comment:
SRAM1 + SRAM2 looks like 256k to me

Reply:
and SRAM3 is 32k
![image](https://user-images.githubusercontent.com/831867/105775549-145b8f80-5fbb-11eb-92fd-8dda95dd91ad.png)
please double check I have that right

Reply:
Yes I agree

Reply:
I've fixed SRAM1+SRAM2. I've left SRAM3 as the second half is not locked, we can change it after the upgrade to ChibiOS 20.3